master/buildbot/test/fake/latent.py

```diff
@@ -216,3 +212,18 @@ def stop_instance(self, build):
             return True
         self._controller._stop_deferred = defer.Deferred()
         return (yield self._controller._stop_deferred)
+
+
+class ControllableLatentWorker(ControllableLatentWorkerMixin,
+                               AbstractLatentWorker):
+    def checkConfig(self, name, _, **kwargs):
+        AbstractLatentWorker.checkConfig(
+            self, name, None,
+            build_wait_timeout=self._controller.build_wait_timeout,
+            **kwargs)
+
+    def reconfigService(self, name, _, **kwargs):
+        AbstractLatentWorker.reconfigService(
+            self, name, None,
+            build_wait_timeout=self._controller.build_wait_timeout,
+            **kwargs)
```

Comment:
same remark about super:

```python
super().checkConfig(name, None, build_wait_timeout=self._controller.build_wait_timeout, **kwargs)
```

and 

```python
super().reconfigService(name, None, build_wait_timeout=self._controller.build_wait_timeout, **kwargs)
```

Reply:
Whoops, the commit is from before we dropped python 2. Fixed now.